### PR TITLE
doc(youtube analytics): dbt source README edit for Fivetran style

### PR DESCRIPTION
**Pull request by fivetran-chloe**
Links https://fivetran.atlassian.net/browse/RD-126487

Edits for Fivetran style. Rewords for clarity. Please address my question (inline) and check my edits for technical accuracy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,12 +39,15 @@ vars:
...
 
 ### Enable Video Metadata
-By default the video metadata functionality within this package is disabled. If you have successfully configured the cloud function above to sync your video metadata into the `videos` table then you will want to enable the package to incorporate the metadata into your package. You may use the below variable configuration in your `dbt_project.yml` to enable this functionality:
+
+By default, the video metadata functionality within this package is disabled. If you have [configured a cloud function to sync your video metadata into a `VIDEOS` table](https://github.com/fivetran/dbt_youtube_analytics_source/blob/main/README.md#youtube-video-metadata), you must enable the package to incorporate the metadata into your package. You may use the variable configuration below in your `dbt_project.yml` to enable this functionality:
```
Comment (fivetran-chloe) [inline README.md]: @fivetran-joemarkiewicz What do you mean by "you must enable the package to incorporate the metadata into your package"? Isn't the customer already enabling the source package?

File: README.md [hunk 2]
```diff
@@ -39,12 +39,15 @@ vars:
...
+The Fivetran YouTube Analytics connector currently does not support video metadata. Consequently, it may be difficult to analyze individual video data without knowing which video belongs to which record. 
+
+As a workaround, you can create a [Functions connector](https://fivetran.com/docs/functions) that syncs your YouTube video metadata into a table in your destination. This dbt package can then use the `VIDEOS` metadata table to enrich your YouTube Analytics reporting data. To learn more about creating a Functions connector, read our [YouTube Analytics Video Metadata Cloud Function article](to_be_created_write-up_link_here). It provides code and detailed steps on how to configure the function.
```
Comment (fivetran-chloe) [inline README.md:45]: Please update this link when the one-pager is ready

File: README.md [hunk 3]
```diff
@@ -54,6 +57,7 @@ vars:
 ```
 
 ### Video Metadata Schema Configuration
+
 By default, this package will look for your `videos` YouTube Analytics metadata table in the `youtube_metadata_schema` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your YouTube Analytics metadata table is, please add the following configuration to your `dbt_project.yml` file:
```
Comment (fivetran-chloe) [inline README.md]:
```suggestion
By default, this package will look for your `VIDEOS` YouTube Analytics metadata table in the `youtube_metadata_schema` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your YouTube Analytics metadata table is, please add the following configuration to your `dbt_project.yml` file:
```

File: README.md (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply README.md]: @fivetran-chloe you are correct the customer is already enabling the source package. This is a result of my poor grammar. I meant you must enable the video metadata functionality within the package, not enable the package itself.

Comment (fivetran-chloe) [reply README.md]: Ah, gotcha - I'll reword to make that clear. Thanks for clarifying!

Comment (fivetran-chloe) [inline README.md]:
```suggestion
By default, the video metadata functionality within this package is disabled. If you have [configured a cloud function to sync your video metadata into a `VIDEOS` table](https://github.com/fivetran/dbt_youtube_analytics_source/blob/main/README.md#youtube-video-metadata), you must enable the video metadata functionality to incorporate the metadata into your package. You may use the variable configuration below in your `dbt_project.yml` to enable this functionality:
```

File: README.md (hunk 2 above)
Comment (fivetran-joemarkiewicz) [reply README.md:45]: Will do, should be done by EOW!

**Comment by fivetran-joemarkiewicz** [comment]
@fivetran-chloe thanks for reviewing the README. I just took a look and all your edits are great and I will move forward with merging to the `main` branch.

Thanks again 🎉

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM - Thanks! 🎉

**Pull request merged**